### PR TITLE
[pretty] Add pretty typing

### PR DESCRIPTION
**Pull request by azerella**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

Adding some type defs for https://github.com/jonschlinkert/pretty ❤️

### CODE REVIEW

File: types/pretty/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+// Project: https://github.com/jonschlinkert/pretty
+// Definitions by:  Adam Zerella <https://github.com/adamzerella>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.3
+
+export interface PrettyOptions {
+    ocd: boolean;
+}
+
+export function pretty(str: string, options?: PrettyOptions): string;
```
Comment (amcasey) [inline types/pretty/index.d.ts]: It looks like the underlying library exports a function.  Should this be `function pretty... ;  export = pretty;`?

File: types/pretty/pretty-tests.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { pretty } from "pretty";
```
Comment (amcasey) [inline types/pretty/pretty-tests.ts]: The underlying library does not appear to export a property called "pretty".  Should this be `import pretty = require("pretty");`?

**Review by amcasey — CHANGES_REQUESTED** [review]

File: types/pretty/pretty-tests.ts (hunk 2 above)
Comment (adamzerella) [reply types/pretty/pretty-tests.ts]: @amcasey Good pickup, I overlooked the testing requirements - fixed.

File: types/pretty/index.d.ts (hunk 1 above)
Comment (adamzerella) [reply types/pretty/index.d.ts]: @amcasey Yeah, I'm still picking up the best way to define type defs here, your right. Fixed

**Review by amcasey — APPROVED** [review]

**Comment by amcasey** [comment]
Thanks for the fixes!

**Pull request merged**